openreview/openreview.py

```diff
@@ -207,6 +208,44 @@ def get_profiles(self, email_or_id_list):
             return { p['email'] : Note.from_json(p['profile'])
                             for p in response.json()['profiles'] }
 
+    def get_pdf(self, id):
+        '''
+        Returns the binary content of a pdf using the provided note id
+        If the pdf is not found then this returns an error message with "status":404
+
+        Example Usage:
+
+        >>> f = get_pdf(id='Place Note-ID here')
+        >>> with open('output.pdf','wb') as op: op.write(f)
+        
+        '''
+        params = {}
+        params['id'] = id
+
+        headers = self.headers.copy()
+        headers['content-type'] = 'application/pdf'
+
+        response = requests.get(self.pdf_url, params = params, headers = headers)
+
+        return response.content
+
+    def put_pdf(self, fname):
+        '''
+        Uploads a pdf to the openreview server and returns a relative url for the uploaded pdf
+
+        :arg fname: path to the pdf
+        '''
+        params = {}
+        params['id'] = id
+
+        headers = self.headers.copy()
+        headers['content-type'] = 'application/pdf'
+
+        with open(fname) as f:
+            response = requests.put(self.pdf_url, files={'data': f}, headers = headers)
+
+        return response.content.strip("{}").split(':')[1].strip('\"')
```

Comment:
I can see why this works, but it might be a little clearer if instead you convert the response content to a dict (like so: `response_dict = json.loads(response.content)`) and then return `response_dict['url']`

@melisabok what do you think?

Reply:
Thanks for the suggestion Michael. Implemented this.